dependencies.yaml

```diff
@@ -181,10 +181,10 @@ channels:
   - rapidsai
   - rapidsai-nightly
   - dask/label/dev
-  - conda-forge
-  - nvidia
   - pytorch
   - dglteam/label/cu118
```

Comment:
It concerns me that we have a channel pointing to CUDA 11.8. This will need to be considered in #3456. cc: @rlratzel

Reply:
Just found out this PR as #3693 was mentioned. 

`cugraph-dgl` package requires `dgl` from `dglteam/label/cu118` channel for testing purposes. That channel/label only consists of the `dgl` package, whose dependencies are surprisingly simple:
```
$ conda search dgl=1.1.1 -c dglteam/label/cu118 --info

Loading channels: done
dgl 1.1.1.cu118 py310_0
-----------------------
file name   : dgl-1.1.1.cu118-py310_0.tar.bz2
name        : dgl
version     : 1.1.1.cu118
build       : py310_0
build number: 0
size        : 71.1 MB
license     : Apache
subdir      : linux-64
url         : https://conda.anaconda.org/dglteam/label/cu118/linux-64/dgl-1.1.1.cu118-py310_0.tar.bz2
md5         : 9f4c8315366dc3091e5e92aade891388
timestamp   : 2023-06-26 05:45:51 UTC
dependencies: 
  - networkx
  - numpy
  - psutil
  - python >=3.10,<3.11.0a0
  - requests
  - scipy
  - tqdm
```

Hopefully it won't collide with cuda packages from `conda-forge` channel.